charts/cactus-backend/values.yaml

```
@@ -126,7 +151,7 @@ persistentVolume:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: ""
+  # storageClass: "gp2"
```

```suggestion
  # storageClass: ""
```
We probably shouldn't set this by default